databases/week3/MealSharingQueries.sql

```diff
@@ -0,0 +1,117 @@
+USE MealSharing;
+
+--Get all meals
+SELECT *
```

Comment:
Just as a side note, it may sometimes be preferable to write out the name of each column so that the result always stays the same, even with the addition of new columns.